scrapers/CzechVR.yml

```diff
@@ -6,31 +6,57 @@ sceneByURL:
       - czechvrcasting.com
       - czechvrfetish.com
     scraper: sceneScraper
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - czechvr.com
+      - czechvrcasting.com
+      - czechvrfetish.com
+    scraper: movieScraper
+
 xPathScrapers:
   sceneScraper:
     common:
-      $info: //div[@class="post"]
-      $url: //meta[@name="dl8-customization-brand-url"]/@content
+      $info: &infoSel //div[@class="post"]
+      $url: &urlSel //meta[@name="dl8-customization-brand-url"]/@content
     scene:
-      Title: $info//div[@class="nazev"]/h2/text()
-      Date:
+      Title: &titleSel
+        selector: $info//div[@class="nazev"]/h2//text()
+        concat: " "
```

Comment:
If you want to keep the studio name in the title a simple
```yaml
     Title: &titleSel $info//div[@class="nazev"]/h2
```
is enough

Reply:
Now CZechVR.com has changed. The $info class changed from "post" to "post home". The details class changed from "textDetail" to "text"... etc. Now I know it's not so easy to keep up with them.

Reply:
Thank you bnkai for your insights ! Learned quite a lot from you!

Reply:
I guess I will redo the CZechVR.yml and see how long it will last this time.

Reply:
Aaah, now I see. The website was not changed. I was simply in the home page instead of a video's page. They looks quite similar. 😃 
Yup, simpler is way better. My understanding of XPath is still too fresh and inexperienced.